### ISSUE TITLE
[BUG] Plugin Configuration Seems to Be Ignored at Times

### ISSUE THREAD

**Issue by Bjoelin** [lochmueller/calendarize#342]
I usually limit the amount of items in a calendarize list view by creating a new *plugin configuration* (which ships with the extension) and selecting one or more event categories within that configuration.

![plugin-config](https://user-images.githubusercontent.com/11597138/69173753-b272e080-0b00-11ea-839d-998d55c24032.jpg)

I then assign the newly created plugin configuration to a plugin content element (CE) which is responsible for rendering a list view. 

**The Problem**

Creating a new plugin config and assigning it to a plugin CE produces the correct frontend output as in a correct amount of items. In my given example (sports events) this amounts to 2 events which is correct - but this works only for a short while.

For some reason (I guess this is somehow cache-related) I am displayed the entirety of _all_ for available events regardless of the category after refreshing the page a couple of times. 

Does anybody know what this might be related to or how to track down where the error stems from? To me, it seems as if the plugin config is "ignored" although it is assigned to a plugin CE.

**Comment by DavidBruchmann** [comment]
You're glad seeing the categories somehow working for short time at least as I opened an issue about categories and got told, it's not implemented

**Comment by Bjoelin** [comment]
@DavidBruchmann Thanks for mentioning the open issue. 

That's confusing, what's the plugin configuration for, then?

**Comment by lochmueller** [comment]
Hey @Bjoelin 
I solced this problem. This is a conceptual problem. The slots reduce the possible items by category and have to result in a item list with valid IDs. If there are no IDs, all items are selected. So your case was possible "There was no events in the selcted category anymore, so all events are displayed". I solced this problem by adding a "-1" to the list, so the list is not empty.
Regards,
Tim
